### ISSUE TITLE
Cannot install: "sh: hook_install: command not found" and "sh: hook_uninstall: command not found"

### ISSUE THREAD

**Issue by marcellodesales** [nodesecurity/nsp-api#2]
I'm trying to use this for the first time and I did "npm install" and it failed with the following... 

```
sh: hook_install: command not found
npm ERR! Linux 2.6.32-431.1.2.el6.x86_64
npm ERR! argv "node" "/usr/bin/npm" "install"
npm ERR! node v0.10.32
npm ERR! npm  v2.1.17
npm ERR! code ELIFECYCLE

npm ERR! precommit-hook@1.0.7 install: `hook_install`
npm ERR! Exit status 127
npm ERR! 
npm ERR! Failed at the precommit-hook@1.0.7 install script 'hook_install'.
npm ERR! This is most likely a problem with the precommit-hook package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     hook_install
npm ERR! You can get their info via:
npm ERR!     npm owner ls precommit-hook
npm ERR! There is likely additional logging output above.

> precommit-hook@1.0.7 uninstall /app_local/github/nsp-api/node_modules/precommit-hook
> hook_uninstall

sh: hook_uninstall: command not found
npm WARN continuing anyway precommit-hook@1.0.7 uninstall: `hook_uninstall`
npm WARN continuing anyway Exit status 127

npm ERR! Please include the following file with any support request:
npm ERR!     /app_local/github/nsp-api/npm-debug.log
```

Thanks!

**Comment by diasdavid** [comment]
`precommit-hook` is a dev dependency (https://github.com/nodesecurity/nsp-api/blob/master/package.json#L48), are you trying to `require('nsp-api')` or to modify this module?

**Comment by marcellodesales** [comment]
Yeah... I need to integrate it with my gulp tasks to validate the current project's package.json... It blows up while I tried to depend on it...

**Comment by diasdavid** [comment]
I'm trying to understand what could be causing the problem, because since precommit-hook is a dev dependency, it is never installed when someone requires it, this is `npm i nsp-api --save` to their project.

Do you have any problems using [`nsp`](https://github.com/nodesecurity/nsp), it also uses it as a dependency

**Comment by marcellodesales** [comment]
Yeah, I know... but some how the "npm install" command is triggering the installation of the hooks :S

**Comment by diasdavid** [comment]
This is the first time I see this behavior and I've been using precommit-hook in dev, production projects (as dev dependencies) and even running in systems I don't control for CI (travis) and never had a issue, can you help me figure out by providing information you might think it can contribute to me figuring it out? It almost feels that problem is that the machine doesn't have git and therefore can't do hooks.. @nlf do you ever ran into this issue?

**Comment by nlf** [comment]
I have not.. Like you said, if you're installing this package as a dependency of your own package then the devDependencies shouldn't even be getting pulled in. @marcellodesales are you sure you're installing the latest version of `nsp-api`? Maybe an old release had `precommit-hook` in the regular dependencies?
